sources/reference/tutorials/batch.md

```diff
@@ -0,0 +1,606 @@
+internal traffic within AWS, amongst various other things. A question we're
+asked a lot is `can I run Chaos Toolkit from AWS, to run against AWS?`.
+The answer is simply, yes, you can.
+
+## Why Batch?
+
+You may have followed our [Running Chaos Toolkit from an EC2 instance][] guide
+and wonder why we would write a guide for Batch - Batch has some benefits
+over EC2:
+
+* Your infrastructure isn't running all the time
+* You can use Docker images to encapsulate your experiment environment
+* You can submit multiple jobs to run different experiments rather than blocking
+on one experiment in an EC2 instance
+
+## Why not ECS and Fargate?
+
+We sometimes get asked about how to run Chaos Toolkit **on** ECS with Fargate,
+whilst we understand why you might want to do this, Chaos Toolkit experiments
+aren't analogous to something like a microservice. We don't run Chaos Toolkit
+continuously and request it to run jobs, rather we invoke Chaos Toolkit when we
+want to use it.
+
+Because of this difference in thinking, we recommend you use Batch (With Fargate
+as the compute provider) to invoke Chaos Toolkit experiments.
+
+## The Steps
+
+For the purposes of this guide, we'll run you through setting up your Chaos Toolkit
+experiments manually. If however, you're familiar with [the AWS Cloud Development
+Kit (CDK)][], we have an example repository deploying the same infrastructure
+using CDK [here][].
+
+[the AWS Cloud Development Kit (CDK)]: https://docs.aws.amazon.com/cdk/latest/guide/home.html
+
+[here]: https://github.com/chaostoolkit/chaostoolkit-aws-batch-example
+
+There are a few pre-requisites required to be able to follow this guide:
+
+* You'll need access to the AWS Console (We're assuming you're comfortable here)
+* You'll need [AWS CLI][] installed and configured
+* You'll need to be able to create EC2 instances (Or have someone do this for
+you)
+* You'll need to be able to create IAM Roles and Policies (Or have someone do
+this for you)
+* You'll need to be able to create Batch Compute Environments, Jobs, and Queues
+(Or have someone do this for you)
+* You'll need to be able to create ECR Repositories and push to them (Or have
+someone to do this for you)
+
+[AWS CLI]: https://aws.amazon.com/cli/
+
+### 1. Create your system (an EC2 instance)
+
+Similar to our [Running Chaos Toolkit from an EC2 instance][] guide, we'll be
+using an EC2 instance as our 'system' to run our experiment against. We'll setup
+our SSH to ensure that the EC2 instance is in a `running` state.
+
+[Running Chaos Toolkit from an EC2 instance]: ../ec2/
+
+* Navigate to the EC2 console and select `Launch Instance`
+* For this guide, we'll select the `Amazon Linux 2 AMI` at the top of the list
+* For this guide, we'll select a `t2.micro` (But you can choose a larger one)
+* Go onto `Configure Instance Details`
+* Select the VPC to deploy into via the `Network` dropdown
+* Select the Subnet to deploy into via the `Subnet` dropdown
```

Comment:
I know we are assuming that the user has already setup a new IAM role via the previous tutorial but should we make reference to it again - just a thought?

Reply:
No assumptions - They don't need it in this tutorial

Reply:
Ah okay, perfect